core/src/model/Entity.java

```diff
@@ -48,9 +55,6 @@ public void setMoving(boolean moving) {
         this.inMotion = moving;
     }
 
-    protected void setEntityName(String name){
```

Comment:
Why is this removed?

Reply:
Because the entity name does not need to be changed.